projects/schematics/src/migrations/4_0/constructor-deprecations/data/login-register.component.migration.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  CHECKOUT_CONFIG_SERVICE,
+  LOGIN_REGISTER_COMPONENT,
+  SPARTACUS_CORE,
+  SPARTACUS_USER,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const LOGIN_REGISTER_COMPONENT_MIGRATION: ConstructorDeprecation = {
+  // feature-libs/user/account/components/login-register/login-register.component.ts
+  class: LOGIN_REGISTER_COMPONENT,
+  importPath: SPARTACUS_USER,
+  deprecatedParams: [],
```

Comment:
In the deprecatedParams you must specify all parameters the constructor currently have.

Reply:
Fixed.